blocks/card-list-block/features/card-list/default.jsx

```diff
@@ -39,6 +39,8 @@ const HeadlineText = styled.h2`
   font-family: ${(props) => props.primaryFont};
 `;
 
+HeadlineText.displayName = 'h2';
```

Comment:
@ktran-wizeline Is this just for having a selector for testing?

Reply:
Yes, you're right

Reply:
couldn't we target the element by its class?

Reply:
Yes, we can do it. But in that case our selector will be **styled.h2**

Reply:
that's fine. you can update the tests with that selector

Reply:
https://hackernoon.com/how-displayname-helps-in-testing-react-using-styled-components-ouad24al

Reply:
I tried with styled.h2 but it's not working, sorry about the confusion because I still see elements rendered with displayName styled.h2 when debuging.